src/index.js

```diff
@@ -121,4 +130,67 @@ function populateManagerDash() {
   availableRooms.innerText = getNumberAvailableRooms().length; 
   revenueToday.innerText = `$${getTodaysRevenue()}`; 
   roomOccupancy.innerText = `${getTodaysOccupancy()}%`;
-}
\ No newline at end of file
+}
+
+//Manager dash right side
+
+function findMatchingUser() {
+  const searchTerm = document.getElementById('search-bar').value;
+  const userToDisplay = userRepo.findUser(searchTerm); 
+  const searchBarError = document.getElementById('no-user-error');
+  if (userToDisplay === undefined) {
+    displayNoUserFoundError(searchBarError);
+  } else {
+    displaySearchResultBox(searchBarError);
+    generateInfoToDisplay(userToDisplay)
+  }
+}
+
+function displayNoUserFoundError(searchBarError) {
+  searchBarError.innerText = 'No user found. Please try again.';
+  const searchResults = document.querySelector('.search-results-display');
+  searchResults.classList.add('hidden');
+}
+
+function displaySearchResultBox(searchBarError) {
+  searchBarError.innerText = '';
+  const searchResults = document.querySelector('.search-results-display');
+  searchResults.classList.remove('hidden');
+}
+
+function generateInfoToDisplay(user) {
+  const userBookings = bookingRepo.getUserBookings(user.id);
+  const userTotalSpent = calculateTotalUserSpend(userBookings);
+  const bookingsHTML = generateBookingsList(userBookings);
+  displayUserInformation(user, userTotalSpent, bookingsHTML)
+}
+
+function displayUserInformation(user, userTotalSpent, bookingsHTML) {
+  const name = document.getElementById('user-name');
+  name.innerText = user.name;
+  const totalSpent = document.getElementById('total-spent-user');
+  totalSpent.innerText = userTotalSpent;
+  const bookingsList = document.getElementById('bookings-list');
+  bookingsList.innerHTML = bookingsHTML; 
+}
+
+function calculateTotalUserSpend(userBookings) {
+  const roomsBooked = roomRepo.getRoomsFromBookings(userBookings);
+  return roomRepo.calculateTotalCost(roomsBooked);
+}
+
+function generateBookingsList(bookings) {
+  const sortedBookings = bookingRepo.sortBookingsByDate(bookings);
+  // today = "2020/02/06";
+  return sortedBookings.reduce((bookingsHTML, booking) => {
+    if (booking.date < today) {
```

Comment:
I love this! Adding the button right next to each booking to delete is a cool feature and very easy for the user to navigate!

Reply:
Thank you!